modules/api/src/main/java/com/intuit/wasabi/api/ApiAnnotations.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright 2017 Intuit
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.intuit.wasabi.api;
+
+public class ApiAnnotations {
```

Comment:
@abhishekjain88: any reason not to use an interface instead of the class? This way you wouldn't need to explicitly put "public static final" in front of each constant as interfaces always have "public static final" fields